src/main/java/io/neonbee/hook/internal/DefaultHookRegistry.java

```diff
@@ -67,7 +67,7 @@ public DefaultHookRegistry(Vertx vertx) {
     @Override
     public CompositeFuture executeHooks(HookType type, Map<String, Object> parameters) {
         List<Future<Void>> hookExecutions = hookRegistry.getOrDefault(type, List.of()).stream()
-                .map(DefaultHookRegistration.class::cast).map(registration -> executeHook(NeonBee.instance(vertx),
+                .map(DefaultHookRegistration.class::cast).map(registration -> executeHook(NeonBee.get(vertx),
                         registration, DefaultHookContext.of(type, parameters)))
                 .collect(Collectors.toList());
 
```

Comment:
unrelated: @line 79
instead of writing `hookRegistry.entrySet().stream().map(Map.Entry::getValue)`
you can replace it with  `hookRegistry.values().stream()`